jupyter-js-widgets/src/widget_selection.js

```diff
@@ -60,6 +61,37 @@ var DropdownView = widget.DOMWidgetView.extend({
         this.update();
     },
 
+    /**
+     * Show the dropdown list.
+     *
+     * If the dropdown list doesn't fit below the dropdown label, this will
+     * cause the dropdown to be dropped "up".
+     * @param  {Event} e
+     */
+    _showDropdown: function(e) {
+        // Don't allow bootstrap to show the dropdown!
+        e.stopImmediatePropagation();
+        e.preventDefault();
+
+        // Get the bottom of the dropdown label, and the bottom of the nb site.
+        // The difference is the maximum height of the dropmenu when displayed
+        // below the button.
+        var droplabelRect = this.$droplabel[0].getBoundingClientRect();
+        var siteRect = document.querySelector('#site').getBoundingClientRect();
```

Comment:
Thanks for catching this BTW!  It's easy for me to slip on things like this.  We should probably have some better non-notebook unit tests to prevent this....